src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/MutingComposter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2023 NotEnoughUpdates contributors
+ *
+ * This file is part of NotEnoughUpdates.
+ *
+ * NotEnoughUpdates is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * NotEnoughUpdates is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.moulberry.notenoughupdates.miscfeatures;
+
+import io.github.moulberry.notenoughupdates.NotEnoughUpdates;
+import io.github.moulberry.notenoughupdates.autosubscribe.NEUAutoSubscribe;
+import io.github.moulberry.notenoughupdates.util.SBInfo;
+import net.minecraftforge.client.event.sound.PlaySoundEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.Arrays;
+import java.util.List;
+
+@NEUAutoSubscribe
+public class MutingComposter {
+
+	private static final MutingComposter INSTANCE = new MutingComposter();
+	private final List<String> mutableSounds = Arrays.asList(
+		"mob.wolf.growl",
+		"tile.piston.out",
+		"liquid.water",
+		"mob.chicken.plop"
+	);
+
+	public static MutingComposter getInstance() {
+		return INSTANCE;
+	}
+
+	protected boolean isEnabled() {
+		return SBInfo.getInstance().getLocation().equals("garden")
+			&& NotEnoughUpdates.INSTANCE.config.garden.muteComposterSounds;
+	}
+
+	@SubscribeEvent
+	public void onSoundPlay(PlaySoundEvent event) {
+		if (mutableSounds.contains(event.name) && isEnabled()) {
```

Comment:
This could also get filtered by checking `event.sound.getXYZPosF()` maybe?

Reply:
I added it, let's hope that hypixel won't do some trolling and they won't change coords of the sound